hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/freon/TestContentGenerator.java

```diff
@@ -57,7 +61,10 @@ public void writeWithByteLevelWrite() throws IOException {
 
   @Test
   public void writeWithSmallBuffer() throws IOException {
-    ContentGenerator generator = new ContentGenerator(1024, 1024, 10);
+    ContentGenerator generator =
+        new ContentGenerator.Builder().keySize(1024).bufferSize(1024)
+            .copyBufferSize(
+                10).build();
```

Comment:
Redundant newline here.

Reply:
Sorry about that, Ill make the necessary changes !!